### PR TITLE
Created IsBanned() & Updated HTTPs url (www.google.com)

### PR DESCRIPTION
**Pull request by Bulldo344**
for better requests, you need to send stuff to www.google.com/robots.txt and better response times instead of having slower response times with www.google.com; 

Created IsBanned() as a Metafunction 

This is my first ever pull request, so if I did anything wrong or need to correct something; Please tell me and please be respectful about it; I always kept my own changes that I've had in my own GitHub repo; just never transferred them over here.

### CODE REVIEW

File: MainModule/Server/Core/HTTP.lua [hunk 1]
```diff
@@ -38,7 +38,11 @@ return function(Vargs)
 	server.HTTP = {
 		Init = Init;
 		Service = service.HttpService;
+<<<<<<< Updated upstream
```
Comment (P3tray) [inline MainModule/Server/Core/HTTP.lua:41]: What's this?

Seems like you added some jargon to your file.

Everything else is great. Good job.

File: MainModule/Server/Core/HTTP.lua [hunk 2]
```diff
@@ -38,7 +38,11 @@ return function(Vargs)
...
 		HttpEnabled = pcall(service.HttpService.GetAsync, service.HttpService, "https://google.com");
+=======
```
Comment (P3tray) [inline MainModule/Server/Core/HTTP.lua:43]: More jargon

File: MainModule/Server/Core/HTTP.lua [hunk 3]
```diff
@@ -38,7 +38,11 @@ return function(Vargs)
...
+		HttpEnabled = pcall(service.HttpService.GetAsync, service.HttpService, "http://www.google.com/robots.txt");
+>>>>>>> Stashed changes
```
Comment (P3tray) [inline MainModule/Server/Core/HTTP.lua:45]: Yet more jargon

**Review by P3tray — CHANGES_REQUESTED** [review]
Just 3 bits of jargon.

File: MainModule/Server/Core/HTTP.lua (hunk 3 above)
Comment (Bulldo344) [reply MainModule/Server/Core/HTTP.lua:45]: That is by Github, not me. Lemme fix that right quick

File: MainModule/Server/Core/HTTP.lua [hunk 4]
```diff
@@ -38,7 +38,11 @@ return function(Vargs)
 	server.HTTP = {
 		Init = Init;
 		Service = service.HttpService;
+<<<<<<< Updated upstream
 		HttpEnabled = pcall(service.HttpService.GetAsync, service.HttpService, "https://google.com");
+=======
+		HttpEnabled = pcall(service.HttpService.GetAsync, service.HttpService, "http://www.google.com/robots.txt");
```
Comment (P3tray) [inline MainModule/Server/Core/HTTP.lua:44]: Wouldn't `1.1.1.1` make more sense? It's resolve speed could be theoretically faster.

Nonetheless requesting robots.txt is significantly faster.

**Review by P3tray — CHANGES_REQUESTED** [review]

File: MainModule/Server/Core/HTTP.lua (hunk 4 above)
Comment (Bulldo344) [reply MainModule/Server/Core/HTTP.lua:44]: It's easier for us to call www.google.com/robots.txt then ``1.1.1.1`` because of roblox issues.

**Comment by Bulldo344** [comment]
Okay, is that good?

File: MainModule/Server/Core/HTTP.lua (hunk 4 above)
Comment (Sceleratis) [reply MainModule/Server/Core/HTTP.lua:41]: thonk

**Review by Sceleratis — CHANGES_REQUESTED** [review]
server.Admin.Banned is not a thing.

**Comment by Bulldo344** [comment]
I'll update whenever I get a chance to.

**Comment by Bulldo344** [comment]
There you go, fixed now.

**Comment by Sceleratis** [comment]
Stuff was borked and I borked it more so manually doing the http change, adding to credits, and closing

**Pull request closed without merging**